pkg/master/server/server.go

```diff
@@ -205,7 +205,18 @@ func (s *APIServer) Run(_ []string) error {
 		glog.Fatalf("Failure to start kubelet client: %v", err)
 	}
 
-	_, v1beta3 := s.RuntimeConfig["api/v1beta3"]
+        //#4544 - screeley44 - test
+        //avoid inadvertent white spaces in the apiserver config file
+        //i.e. KUBE_API_ARGS="--runtime_config=api/v1beta3  "
+        v1beta3 := false
+        for k, _ := range s.RuntimeConfig {
```

Comment:
Thanks, but this isn't the right "fix" for this problem. It needs to be addressed in pkg/util/configuration_map.go, by banning whitespace in key names.